package.json

```
@@ -3,7 +3,7 @@
     "displayName": "Fix All JSON",
     "description": "Fix missing and trailing commas on save",
     "publisher": "zardoy",
-    "version": "0.0.0-dev",
+    "version": "0.0.1",
```

```suggestion
    "version": "0.0.0-dev",
```

Version managed automatically, revert this one. The version [is set](https://github.com/zardoy/release-action/blob/b877a561231a199a7c48e849b8561196f5272207/src/library/main.ts#L112) at publishing.

This is actually a common practice for guys using [semantic-release like tools](https://github.com/semantic-release/semantic-release/blob/27b105337b16dfdffb0dfa36d1178015e7ba68a3/package.json#L4)